dubbo-spring-boot/dubbo-spring-boot-actuator/src/main/resources/META-INF/dubbo-endpoints-default.properties

```diff
@@ -2,20 +2,58 @@
 # those values of properties can be override by higher PropertySource
 # @see DubboEndpointsAutoConfiguration
 
-# Set enabled for Dubbo Endpoints
-management.endpoint.dubbo.enabled = true
-management.endpoint.dubboshutdown.enabled = false
-management.endpoint.dubboconfigs.enabled = true
-management.endpoint.dubboservices.enabled = false
-management.endpoint.dubboreferences.enabled = false
-management.endpoint.dubboproperties.enabled = true
-
 # "management.endpoints.web.base-path" should not be configured in this file
-
 # Re-defines path-mapping of Dubbo Web Endpoints
-
 management.endpoints.web.path-mapping.dubboshutdown = dubbo/shutdown
 management.endpoints.web.path-mapping.dubboconfigs = dubbo/configs
 management.endpoints.web.path-mapping.dubboservices = dubbo/services
 management.endpoints.web.path-mapping.dubboreferences = dubbo/references
 management.endpoints.web.path-mapping.dubboproperties = dubbo/properties
+# Set enabled for Dubbo Endpoints
```

Comment:
management.endpoint.dubboofflineapp.enabled
management.endpoint.dubboofflineinterface.enabled
management.endpoint.dubbogracefulshutdown.enabled
These need to default to false.